v2/session/common-customizations/sessions/get-access-token.mdx

```diff
@@ -0,0 +1,405 @@
+---
+id: get-access-token
```

Comment:
i feel like this page will confuse people, especially those that are using supertokens with a JWT. 

Originally, what i meant was that this page would have the same content as the "handling session tokens" page in other recipe with a `show_ui_switcher: true`, and not just about how to get access token.

But im also just happy to get rid of this page along with the other "Getting the access token" pages in other recipes.

Reply:
I've removed this for now - it will make more sense after the JWT rework. This would only be useful to a few people anyway until then.